unstructured/partition/strategies.py

```diff
@@ -72,6 +73,13 @@ def determine_pdf_or_image_strategy(
     logic if some dependencies are not available."""
     pytesseract_installed = dependency_exists("pytesseract")
     unstructured_inference_installed = dependency_exists("unstructured_inference")
+    if not strategy:
+        if torch.cuda.is_available() or (
```

Comment:
@kravetsmic - For this one, the `torch` library is only available if you install the `local-inference` extra, so I think this would fail with just base `unstructured`. Could you wrap this in a `_is_gpu_available` function, and also check first if `torch` is installed?